README.md

```diff
@@ -250,6 +251,26 @@ import "github.com/open-feature/go-sdk/openfeature"
 openfeature.Shutdown()
 ```
 
+
+### Transaction Context Propagation
+
+Transaction context is a container for transaction-specific evaluation context (e.g. user id, user agent, IP).
+Transaction context can be set where specific data is available (e.g. an auth service or request handler) and by using the transaction context propagator it will automatically be applied to all flag evaluations within a transaction (e.g. a request or thread).
+
+```go
+import "github.com/open-feature/go-sdk/openfeature"
+
+// set the TransactionContext
+ctx := openfeature.WithTransactionContext(context.Background(), openfeature.EvaluationContext{})
+
+// get the TransactionContext from a context
+ec := openfeature.TransactionContext(ctx)
+
+// merge an EvaluationContext with the existing TransactionContext, preferring
+// the context that is passed to MergeTransactionContext
+ctx := openfeature.MergeTransactionContext(ctx, openfeature.EvaluationContext{})
+```
+
```

Comment:
nit - I think it's good to show the derived transaction context usage in a flag evaluation. For example,

 ```go
 // set the TransactionContext
tCtx := openfeature.WithTransactionContext(context.Background(), openfeature.EvaluationContext{})

// use TransactionContext in a flag evaluation
client.BooleanValue(tCtx, ....) 
 ```